test/integration/storage.spec.ts

```diff
@@ -38,6 +38,25 @@ describe('admin.storage', () => {
     return verifyBucket(bucket, 'storage().bucket(string)')
       .should.eventually.be.fulfilled;
   });
+  it('getDownloadUrl returns a download URL', async () => {
+    const bucket = getStorage().bucket(projectId + '.appspot.com');
+    const destination = 'test_files/test.txt';
+    const [fileRef] = await bucket.upload('./test/integration/test.txt', {
```

Comment:
Can we do something similar to what we do in `verifyBucket()` (line 51) here instead of uploading a file?
```
  const expected: string = 'Hello World: ' + testName;
  const file: File = bucket.file('data_' + Date.now() + '.txt');
  file.save(expected)
```